test/demo/service.yaml

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: demo-service
+  labels:
+    app: demo
+spec:
+  type: NodePort
+  ports:
+  - nodePort: 31000
+    port: 8080
+    name: demo-port
+    targetPort: 8080
```

Comment:
targetPort is defined automatically by the system, so we don't need to specify it.


This works for me:
```
--- a/test/demo/service.yaml
+++ b/test/demo/service.yaml
@@ -10,9 +10,7 @@ metadata:
 spec:
   type: NodePort
   ports:
-  - nodePort: 31000
-    port: 8080
-    name: demo-port
-    targetPort: 8080
+    - port: 80
+      name: demo-port
   selector:
     app: demo
```